apps/src/music/player/interfaces/PlaybackEvent.ts

```diff
@@ -21,4 +21,6 @@ export interface PlaybackEvent {
   length: number;
   /** The ID of the block that created this event */
   blockId?: string;
+  /** A unique ID used to group same sounds together in the timeline */
+  id?: string;
```

Comment:
I came across a situation in which it wasn't needed when playing a preview.

Reply:
Oh gotcha. I think it might still be a good idea to make it required so that we can trust that an event will always have an ID of some kind (even if the Timeline view isn't in TS)? For the previews maybe we can just pass an ID of 'preview' since we know it'll be ignored?

Reply:
Good call.  Have updated.